deploy/vagrant/provision/roles/tachyon/files/set_journal_folder.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# For Tachyon version that is no less than 0.8
+echo 'tachyon.master.journal.folder=${tachyon.underfs.address}/tachyon/journal/' >> /tachyon/conf/tachyon-site.properties
+
+# For earlier Tachyon versions
+sed -i "s/tachyon.master.journal.folder=.*/tachyon.master.journal.folder=\$TACHYON_UNDERFS_ADDRESS\/tachyon\/journal\//g" /tachyon/conf/tachyon-env.sh
```

Comment:
In versions less than 0.8, this property is set and passed through tachyon-env.sh, which will overwrite settings in tachyon-site.properties.